circle.yml

```diff
@@ -17,12 +17,10 @@ test:
     - make docs
 deployment:
     production:
-        branch: master
+        tag: /v*/
         commands:
-            - make docs
             - aws s3 sync ~/sceptre/docs/_build/html s3://sceptre.cloudreach.com/docs/ --delete
```

Comment:
Yup so thats what would happen - only commits with `v*` tags would go to `/docs` and all commits regardless of tags goes to `/dev/docs`